### DISCUSSION TITLE
API "set" case

### DISCUSSION THREAD

**Discussion by yovanoc** [prisma/prisma#2253] [Q&A] [answered]
Hello!

I want to know how we can really "set". I mean if we put some new things it will create, otherwise update (like we can do today), and delete too!

Thanks

**Comment by ryands17** [comment, marked as answer]
Hey @yovanoc :wave: 
So what you're saying is something like the `upsert` functionality along with delete support?

**Reply by yovanoc** [reply]
Exactly! sorry for my explanation!

**Reply by ryands17** [reply]
Awesome! Currently prisma doesn't support this functionality with `delete` support, so can you please create a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) :smile:

**Reply by yovanoc** [reply]
Great! done here: https://github.com/prisma/prisma/issues/2255

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
